back/engines/commercial/idea_custom_fields/spec/acceptance/idea_custom_fields/phase_context/index_spec.rb

```diff
@@ -10,6 +10,7 @@
 
   get 'web_api/v1/admin/phases/:phase_id/custom_fields' do
     parameter :support_free_text_value, 'Only return custom fields that have a freely written textual answer', type: :boolean, required: false
+    parameter :copy, 'Copy this form (resets all fields)', type: :boolean, required: false
```

Comment:
Disambiguation? _'resets all fields'_ refers to form fields or `custom_fields`

Not sure I really understand use of 'reset' either. This creates copies of `custom_fields` with different ids, no?